src/main/scala/com/snowplowanalytics/indicative/LambdaHandler.scala

```diff
@@ -36,6 +37,7 @@ class LambdaHandler {
     sys.env.getOrElse("INDICATIVE_API_KEY",
                       throw new RuntimeException("You must provide environment variable INDICATIVE_API_KEY"))
   val indicativeBatchSize = 100
+  val appIdPattern: Regex = sys.env.getOrElse("APP_ID_REGEX", ".*").r
```

Comment:
If the APP_ID_REGEX env variable doesn't exist then default to ".*" which would match anything.

Reply:
@BenFradet you have a point. I initially wanted to push back in the interest of keeping the code simple, but I ran some tests and unwrapping an Option (like a sys.env.get) is orders of magnitude faster than running a regex. I mean we are still talking milliseconds vs nano seconds but over 100 million iterations the difference really adds up. I think we should use the env variable itself as the option (i.e. if it is blank we don't use it) rather than add an additional environment variable to the mix. 

I might need some help with the best way to implement this in an idiomatic way, Scala is not my first language (or even my 10th lol).